src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -622,20 +624,37 @@ def create_arrange_directory(path):
     )
 
 
+def create_arrange_directories(paths):
+    arranges = [
+        models.SIPArrange(
+            original_path=None,
+            arrange_path=os.path.join(path, ""),  # ensure ends with /
+            file_uuid=None,
+        )
+        for path in paths
+        if path.startswith(DEFAULT_ARRANGE_PATH)
```

Comment:
I think we should probably replicate the error behaviour here (it's more of a pain with multiple but I think you can just do a guard at the start of the function, something like:
```
        for path in paths:
            if not path.startswith(DEFAULT_ARRANGE_PATH):
                raise ValueError(_("All directories must be within the arrange directory.")
```

Reply:
:+1: